src/Symfony/Component/Security/Core/Resources/translations/security.sq.xlf

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="1">
+                <source>An authentication exception occurred.</source>
+                <target>Ndodhi një problem në verifikim.</target>
```

Comment:
Can you check this one too?

Reply:
authentication and verification in Albanian are kind of similar but will update it to match exactly to authentication